amz_review_scraper/soup_searcher.py

```diff
@@ -1,20 +1,59 @@
+product_json = {}
+product_json["details"] = []
 
 
+def find_attribute(soup, key, html_tag, attrs, *args, **kwargs):
```

Comment:
@davidbegin what are your thoughts on removing `attrs` from this and only passing it in as a kwargs.